contrib/systemd/containerized-installs/origin-node.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+After=docker.service
+Requires=docker.service
+Wants=openvswitch.service
+After=origin-master.service
+
+[Service]
+EnvironmentFile=/etc/sysconfig/origin-node
+ExecStartPre=-/usr/bin/docker rm -f origin-node
+ExecStart=/usr/bin/docker run --name origin-node --rm --privileged --net=host --pid=host -v /:/rootfs:ro -e CONFIG_FILE=${CONFIG_FILE} -e OPTIONS=${OPTIONS} -e HOST=/rootfs -v /etc/systemd/system:/host-etc/systemd/system -e HOST_ETC=/host-etc -v /etc/localtime:/etc/localtime:ro -v /etc/machine-id:/etc/machine-id:ro -v /lib/modules:/lib/modules -v /run:/run -v /sys:/sys:ro -v /usr/bin/docker:/usr/bin/docker:ro -v /var/lib/docker:/var/lib/docker -v /etc/origin/node:/etc/origin/node:z -v /etc/origin/openvswitch:/etc/openvswitch:Z -v /etc/origin/sdn:/etc/openshift-sdn:Z -v /var/lib/origin:/var/lib/origin:Z openshift/node
```

Comment:
same comments as above, :z on stuff in /etc/looks scary to me, why need docker rm ?